src/test/resources/uk/gov/justice/hmpps/nomis/datacompliance/repository/jpa/repository/define_jaro_winkler_similarity.sql

```diff
@@ -0,0 +1,9 @@
+-- Mapping the Oracle UTL_MATCH.JARO_WINKLER_SIMILARITY function to a stubbed value:
```

Comment:
HSQLDB has no equivalent for UTL_MATCH.JARO_WINKLER_SIMILARITY so we need to define the function for testing.